docs/api/owner-details-template-resource.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Owner URL Template, NuGet API
+description: The Owner URL template allows clients to display in their UI a web link
+author: martinrrm
+ms.author: mruizmares
+ms.date: 4/15/2024
+ms.topic: reference
+ms.reviewer:
+---
+
+# Owner URL template
+
+It is possible for a client to build a URL that can be used by the user to see owner's page in their web browser. 
+This is useful when a package source wants to show additional information about an owner that may not fit within the scope of what the NuGet client application shows.
+
+The resource used for building this URL is the `OwnerDetailsUriTemplate` resource found in the
+[service index](service-index.md).
+
+## Versioning
+
+The following `@type` values are used:
+
+@type value                     | Notes
+------------------------------- | -----
+OwnerDetailsUriTemplate/6.11.0  | The initial release
+
+## URL template
+
+The URL for the following API is the value of the `@id` property associated with one of the aforementioned resource `@type` values.
+
+## HTTP methods
+
+Although the client is not intended to make requests to the owner URL on behalf of the user, the web page should support the `GET` method to allow a clicked URL to be easily opened in a web browser.
+
+## Construct the URL
+
+Given a known owner ID, the client implementation can construct a URL used to access a web interface.
+The client implementation should display this constructed URL (or clickable link) to the user allowing them to open a web browser to the URL and to learn more about the owner.
+The contents of the owner page is determined by the server implementation.
+
+The URL must be an absolute URL and the scheme (protocol) must be HTTPS.
+
+The value of the `@id` in the service index is a URL string containing any of the following placeholder tokens:
+
+### URL placeholders
+
+Name        | Type    | Required | Notes
+----------- | ------- | -------- | -----
+`{owner}`   | string  | no       | The owner ID to get details for
```

Comment:
There are two kinds of required that matter I think: 1) the client rejects or errors out when no `{owner}` parameter is found and 2) we say it's required in the docs because we think it should really be there, but we make no effort to enforce it in the client. 

I think having the same URL for every owner (i.e. no `{owner}` token) is a bad user experience but I don't know how to best surface that error in client.

In short, I think saying required = yes in the docs as a description of intent is okay even if client does no validation on this. required = no gives the wrong intent

Reply:
"Required yes" sounds good to me. 

I agree that it's a bad user experience to have identical links for all owners, but we could leave that up to the package source. I wanted to know whether to add validation in Client in my work in progress. I will not and just allow it to get the same repeated URL for each owner.